tests/constraints-devel.in

```diff
@@ -0,0 +1,5 @@
+# Known limitations for indirect/transitive dependencies.
+antsibull-docs >= 2.0.0, < 3.0.0
+pygments >= 2.10.0
+sphinx-notfound-page >= 0.6
+sphinx-ansible-theme >= 0.10.2
```

Comment:
These look like direct dependencies, not constraints: the minimums are only useful in the place where you define the direct dependencies. pip attempts to use the latest versions anyway.